TideApp/TideApp/Scenes/TideTimesView/TideTimesInfoView.swift

```diff
@@ -23,21 +24,30 @@ struct TideTimesInfoView: View {
           BodyLabel(text: tideStatus)
             .padding(.bottom, PaddingValues.tiny)
         }
-        ForEach(weatherInfo.tideTimes) { tideTime in
-          BodyLabel(text: "\(tideTime.tideType.rawValue.capitalized): \(tideTime.tideTime)")
+
+        HStack {
+          VStack {
+            ForEach(weatherInfo.tideTimes) { tideTime in
+              BodyLabel(text: "\(tideTime.tideType.rawValue.capitalized): \(tideTime.tideTime)")
+                .padding(.bottom, 15)
```

Comment:
could maybe use `PaddingValues.medium` for this?

Reply:
Yes, will update